docs/data-sources/cloud_accounts.md

```diff
@@ -0,0 +1,35 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "astra_cloud_accounts Data Source - terraform-provider-astra"
+subcategory: ""
+description: |-
+  Retrieve a list of Cloud Accounts within an Organization
+---
+
+# astra_cloud_accounts (Data Source)
+
+Retrieve a list of Cloud Accounts within an Organization
```

Comment:
This is probably more a of a question for the DevOps API folks, but will this only ever return an array of length 1, or are there situations where Astra organizations will have more than one cloud accounts for a given cloud provider and region?

I ask since the primary use case for this data source is to be able to feed it into an AWS KMS access policy, which would require creating an ARN based on the `provider_id` returned in this data source's response. I'm curious if it is safe to just refer to it with `data.astra_cloud_accounts.this[0].provider_id`, or if some HCL manipulation needs to be done to turn it into an array of provider IDs/ARNs for the KMS key policy.

Reply:
For GCP, I happen to have 2 accounts, so it can be more than 1. That may only be the case for GCP, but the code handles both AWS and GCP.